test/acceptance/steps/select-business-step.js

```diff
@@ -36,8 +36,14 @@ When(/^user is on the livestock page$/, async function () {
 When(/^user check if livestock are listed$/, async function () {
   await selectBusinessPage.livestockList()
 })
-When(/^user choose the livestock to review$/, async function () {
-  await selectBusinessPage.livestockToReview()
+When(/^user choose dairy cattle for review$/, async function () {
+  await selectBusinessPage.sheepReview()
+})
+When(/^user choose beef cattle for review$/, async function () {
+  await selectBusinessPage.beefCattleReview ()
+})
+When(/^user choose sheep for review$/, async function () {
```

Comment:
This should be changed to dairy cattle and the other should be changed to sheep.

Reply:
This is a step def, and each map the required feature regardless of the positioning

Reply:
The positioning is wrong. You wrote "user chose sheep for review" and then run the dairy cattle script. This is wrong.